src/node/watch-webpack.js

```diff
@@ -89,26 +89,11 @@ function watchWebpack(
         );
       };
 
-      compiler.watch(
-        {
-          ignored: [
-            /node_modules/,
-            // Ignore temporary files because they are only re-generated by
-            // watchContext, and we don't want to double-compile.
-            path.join(batfishConfig.temporaryDirectory, './**/*')
-          ]
-        },
-        onCompilation
-      );
+      compiler.watch({ ignored: [/node_modules/] }, onCompilation);
 
       // Watch pages separately, so we can rewrite the context module, which
       // will capture changes to front matter, page additions and deletions.
-      watchContext(batfishConfig, {
-        onError: onError,
-        afterCompilation: () => {
-          compiler.run(onCompilation);
```

Comment:
This was the root problem: Webpack no longer allows you to `compiler.run` when the `compiler` is already watching. When I removed this `afterCompilation` run and allowed `compiler.watch` to pick up on changes to the context module in the temporary directory, things seemed to work OK. So I'm hoping that the original motivation for this added complexity no longer applies, with the latest Webpack stuff.